src/modules/combinedFacets/CombinedFacetsContainer.js

```diff
@@ -15,6 +18,24 @@ import GenerateCombinedFacets from './generateCombinedFacets';
 import { executeCallback } from '../../common/utils';
 
 class CombinedFacetsContainer extends React.PureComponent {
+
+    componentDidMount(){
+        const {
+            categoryDisplayName = '',
+            categoryField = '',
+            facetDepth,
+            facetLimit,
+            helpers: { setMultilevelFacetsConfiguration }
+        } = this.props;
+
+        setMultilevelFacetsConfiguration({
+            categoryDisplayName,
```

Comment:
Same here.

Reply:
its used in the setMultilevelFacetsConfiguration

Reply:
But do we require it here?